packages/disputer/src/disputer.js

```diff
@@ -204,7 +200,8 @@ class Disputer {
           message: "Position has been disputed!👮‍♂️",
           liquidation: disputeableLiquidation,
           disputeResult: logResult,
-          totalPaid: returnValue
+          totalPaid: returnValue.toString(),
+          transactionConfig
```

Comment:
Any particular reason to add back `transactionConfig` to logs or just for completeness? I removed because I recall it doesn't print particularly well and sometimes one of the keys in the `transactionConfig` would turn up null, causing the Logger to throw.

Reply:
ye added just for better logging. I guess it's pretty useless seeing we can always get it from etherscan. as long as we have the tx hash I think we can remove this.

Reply:
Actually looking again I remember the main reason I did this. The `runTransaction` method will _modify_ the config passed in, such as changing the `fromAddress`, `nonce`, `usingOffSetDSProxyAccount` and injects the starting `gasPrice`. By having access to this config we can see if the offset was used and additional data that is not visible from etherscan.

Ideally, I think it should be included to help with understanding what's going on when bumping EOA accounts.

Reply:
however, you are right that this will be pretty noisy... not sure exactly. WDYT?

Reply:
These logs are relatively rare, might as well leave it in since we're modifying `transactionConfig` now